deps/chakrashim/core/CONTRIBUTING.md

```diff
@@ -0,0 +1,54 @@
+## Contributing Code
```

Comment:
@jianchun: okay, I didn't know about that convention. I disagree with it, but I guess changing that convention is another discussion. To answer your question, if the upstream needs to be modified without changing e.g. the master branch, a new branch should be created upstream.